src/lib.rs

```diff
@@ -21,13 +23,14 @@ use tokio::{
     io::{AsyncBufReadExt, AsyncWriteExt, BufReader},
     net::{TcpListener, TcpStream},
     sync::RwLock,
+    time::{interval, Duration},
 };
 
 #[derive(Debug)]
 struct ForwardMessage {
     path: String,
     host: IpAddr,
-    port: u32,
+    port: u64,
```

Comment:
I imagine this `port` is a bindable port?

If so, you can probably make it a `u16` since the max value can be `65535` and not have to allocate space for a much larger value with the `u64` that will never be hit

Reply:
Yeah I agree having this as a u16 makes way more sense.